packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx

```diff
@@ -19,9 +19,28 @@ const defaultProps = {
   src: ''
 };
 
-const Avatar = ({ className, src, alt, ...props }) => (
-  <img {...props} className={css(_styles.avatar, className)} src={src} alt={alt} />
-);
+/**
+ * Column properties.
+ */
+export interface IAvatarProps {
+  /** Additional classes added to the Avatar. */
+  className?: string;
+  /** Attribute that specifies the URL of the image for the Avatar. */
+  src?: number;
+  /** Attribute that specifies the alt text of the image for the Avatar. */
+  alt: string
+}
+
+class Avatar extends React.Component<IAvatarProps, {}> {
```

Comment:
This can be `const Avatar: React.SFC<AvatarProps>` instead of using the class.

Reply:
Additionally for classes the second generic (state) defaults to `{}`. So no need to specify it if state is not being used.

Reply:
Done.